src/shared/components/rightbar/RightBar.tsx

```diff
@@ -28,15 +28,10 @@ export default class RightBar extends React.Component<{}, {}> {
 
                 <div className="rightBarSection">
                     <h3>What's New <a href="http://www.twitter.com/cbioportal" className="pull-right">@cbioportal <i className="fa fa-twitter" aria-hidden="true"></i></a></h3>
-                        <p>Sign up for low-volume email news alerts:</p>
-
-                        <form className="form-inline">
-                            <div className="form-group form-group-sm">
-                                <input type="email" className="form-control" id="exampleInputEmail2" placeholder="jane.doe@example.com" />
-                            </div>
-
-                            <button type="submit" className="btn btn-default btn-sm">Subscribe</button>
-                        </form>
+                    <p>Sign up for low-volume email news alerts:</p>
+                    <a target="_blank" href="http://groups.google.com/group/cbioportal-news/boxsubscribe">
+                        <button type="button" className="btn btn-default btn-sm" style={{width: "100%"}}>Subscribe</button>
```

Comment:
we shouldn't wrap a button in an a tag.  not semantically correct.  the bootstrap classes (btn btn-default btn-sm) can actually be assigned to <a> tag (style tag too) and then we can lose the <button> altogether